### PR TITLE
Loading the work to do widget

### PR DESCRIPTION
**Pull request by m6jones**
Changes to load the work to do widget. First page only.

### CODE REVIEW

File: features/workToDo/w2dSummonActionObserver.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class W2dSummonAction extends SirenSummonAction {
 			});
 		}
 		super.addObserver(observer, property, { method, route });
+		this._setPage(observer[page]);
```
Comment (tenbradley) [inline features/workToDo/w2dSummonActionObserver.js:21]: If `observer[page]` is null, do we want to update the page to null?

File: features/workToDo/d2l-w2d-list.js [hunk 2]
```diff
@@ -36,17 +36,9 @@ class W2dList extends HypermediaStateMixin(LitElement) {
...
-		if (this.skeleton || !this._loaded) {
-			return lithtml`
-				<d2l-list separators="${this.collapsed ? 'none' : 'all'}">
-					<d2l-w2d-list-item skeleton ?collapsed="${this.collapsed}"></d2l-w2d-list-item>
-					<d2l-w2d-list-item skeleton ?collapsed="${this.collapsed}"></d2l-w2d-list-item>
-					<d2l-w2d-list-item skeleton ?collapsed="${this.collapsed}"></d2l-w2d-list-item>
-				</d2l-list>
-			`;
-		}
+		if (this.skeleton) return this._renderSkeleton();
```
Comment (tenbradley) [inline features/workToDo/d2l-w2d-list.js:39]: Is there a reason this isn't one if statement?
`if (this.skeleton || !this._activities) return this._renderSkeleton();`

File: features/workToDo/d2l-w2d-list-item.js [hunk 3]
```diff
@@ -169,11 +172,12 @@ class W2DListItemMixin extends HypermediaStateMixin(ListItemLinkMixin(LocalizeDy
 	}
 
 	_renderSkeleton() {
+		if (!this.skeleton && this._loaded) return;
 		return this._renderListItem({
 			illustration: html`<d2l-activity-icon skeleton></d2l-activity-icon>`,
 			content: html`
 				<d2l-list-item-content>
-					<d2l-activity-name class="d2l-w2d-list-item-name" skeleton></d2l-activity-name>
+					<d2l-activity-name class="d2l-w2d-list-item-name" skeleton ></d2l-activity-name>
```
Comment (tenbradley) [inline features/workToDo/d2l-w2d-list-item.js:180]: Is this added space needed?

File: features/workToDo/d2l-w2d-collections.js [hunk 4]
```diff
@@ -22,7 +23,7 @@ const rel = Object.freeze({
 
 const pageSize = Object.freeze({
 	collapsed: 6,
-	fullScreen: 20
+	fullScreen: 3
```
Comment (tenbradley) [inline features/workToDo/d2l-w2d-collections.js:26]: I think you want to change this back to 20?

File: features/workToDo/d2l-w2d-collections.js [hunk 5]
```diff
@@ -326,49 +333,26 @@ class W2dCollections extends LocalizeDynamicMixin(HypermediaStateMixin(LitElemen
...
-
 	get _pageSize() {
 		return this.__pageSize;
 	}
 
 	set _pageSize(pageSize) {
 		const oldValue = this._pageSize;
 		this.__pageSize = pageSize;
-		this._pageOverdue = this._page;
+		//this._pageOverdue = this._page;
```
Comment (tenbradley) [inline features/workToDo/d2l-w2d-collections.js:346]: remove this comment

File: features/workToDo/w2dSummonActionObserver.js (hunk 1 above)
Comment (m6jones) [reply features/workToDo/w2dSummonActionObserver.js:21]: I tend to handle that within the method.

**Review by lloydwaddell — APPROVED** [review]

**Pull request merged**